hazelcast/src/test/java/com/hazelcast/internal/ascii/MemcachedTest.java

```diff
@@ -62,7 +62,6 @@
 
     protected Config createConfig() {
         Config config = smallInstanceConfig();
-        config.setProperty(GroupProperty.MEMCACHE_ENABLED.getName(), "true");
```

Comment:
Since enabling the memcache protocol by property is now removed, we need to explicitly enable memcache in this test by adding `config.getNetworkConfig().getMemcacheProtocolConfig().setEnabled(true);` here.